dftimewolf/lib/state.py

```diff
@@ -300,6 +305,14 @@ def GetContainers(self,
         for c in ret_val:
           self.store[container_class.CONTAINER_TYPE].append(c)
 
+      calling_class = inspect.stack()[1][0].f_locals["self"].__class__.__name__
+      if calling_class != 'DFTimewolfState':
+        verb = 'popping' if pop else 'retrieving'
+        logger.debug(f'{calling_class} is {verb} {len(ret_val)} '
+            f'{container_class.CONTAINER_TYPE} containers:')
```

Comment:
Maybe use a constant sentence with (poppling: True/False) at the end? I think it makes the difference stand out more.

Reply:
Done.